.babelrc

```diff
@@ -1,11 +1,27 @@
 {
-  "plugins": [
-    "transform-runtime",
-    "transform-async-generator-functions"
-  ],
-  "presets": [
-    ["env", {
-      "loose": true
-    }]
-  ]
+  "env": {
+    "es5": {
+      "plugins": [
+        "add-module-exports",
```

Comment:
Writes module.exports = exports['default'] to the end of the transpiled files if there is only a default export.